alerts/ui/src/components/filters/FilterPills.jsx

```diff
@@ -22,6 +42,21 @@ const FilterPills = () => {
             closeable
             onClose={() => removeActiveFilter(key, value)}
             key={`${key}:${value}`}
+            onClick={() => pauseFilter(key, value)}
+          />
+        ))
+      })}
+      {Object.entries(pausedFilter).map(([key, values]) => {
+        console.log("sdfsdf", values)
+        return values.map((value) => (
+          <Pill
+            className="bg-theme-background-lvl-4"
```

Comment:
Instead of adjusting the bg color, try changing the opacity of the paused filters, e.b. `opacity-70`